<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_null_editUserDialog</class>
 <widget class="QDialog" name="dialog_null_editUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_editUserDialog_userName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_editUserDialog_userPassword">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_editUserDialog_userName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_editUserDialog_userpassword">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_editUserDialog_videoManager">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>录像管理</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_editUserDialog_conntionManager">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>链接管理</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_editUserDialog_shutdown">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭系统</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_editUserDialog_yunTaiControl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>云台控制</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_editUserDialog_systemSet">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>系统设置</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_editUserDialog_videoExchange">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>视频调节</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_editUserDialog_capturePicManager">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>抓图管理</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_editUserDialog_userManager">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_editUserDialog_selectAll">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_editUserDialog_noSelectAll">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>全不选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_editUserDialog_sure">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_editUserDialog_cancel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
